src/datasource.js

```diff
@@ -262,10 +283,20 @@ export default class OCIDatasource {
           options.scopedVars
         )}[${window}]${dimension}.${t.aggregation}`;
       }
+      console.log("checkpoint 1")
 
+      const tenancyconfig =
+      this.getVariableValue(t.tenancyconfig, options.scopedVars) === SELECT_PLACEHOLDERS.TENANCYCONFIG
+        ? DEFAULT_TENANCYCONFIG
+        : this.getVariableValue(t.tenancyconfig, options.scopedVars);        
+      let target = {
+        tenancyconfig: tenancyconfig,
+        region: _.isEmpty(region) ? this.defaultRegion : region,
+      }; 
       const compartmentId = await this.getCompartmentId(
-        this.getVariableValue(t.compartment, options.scopedVars)
+        this.getVariableValue(t.compartment, options.scopedVars), target
       );
+      console.log("checkpoint 2")
```

Comment:
I am guessing this will be deleted

Reply:
The code is still WIP, so I need several checkpoints in the logs to test functionalities. Obviously they will disappear in the final version